formats/qcow.nix

```diff
@@ -5,7 +5,10 @@
   modulesPath,
   specialArgs,
   ...
-}: {
+}:
+let
+  contents = specialArgs.qcow.contents or [];
```

Comment:
@Lassulus why we do we now use specialArgs instead of defining cleaner nixos options?

Reply:
I was thinking about this also when making pr for https://github.com/nix-community/nixos-generators/pull/387 Is there a better way of exposing parameters of the upstream imports?

Reply:
Missed this PR and comment before, but just answered in #387: The "parameters" to upstream imports are "module options". You should be able to just set them in an argument to `modules` in `nixosGenerate`, see https://github.com/nix-community/nixos-generators/pull/387#issuecomment-2446293154 for an example of this.
(or #375 for a more elaborate one ;))